### ISSUE TITLE
grunt-accessibility doesn't run and just hangs

### ISSUE THREAD

**Issue by jamesmhaley** [yargalot/grunt-accessibility#41]
Hey guys, 

I've been trying most of the day to get the a grunt script to run to no avail.

```javascript
grunt.initConfig({
    accessibility: {
        options : {
            accessibilityLevel: 'WCAG2A',
            domElement: false,
            force: true
        },
        test : {
            files: [{
                expand  : true,
                cwd     : 'templates',
                src     : ['*.html'],
                dest    : 'reports/',
                ext     : '-report.txt'
            }]
        }
    }
});

grunt.loadNpmTasks('grunt-accessibility');

grunt.registerTask('accessibility', ['accessibility']);
```

The script just hangs, no feedback from grunt, no error. Just hangs...

`grunt accessibility`

I have installed the plugin using: npm install grunt-accessibility --save-dev

Ae their other dependancies? All my other grunt tasks run fine.

Using pa11y/a11y runs fine, just this plugin doesn't.

It just sit's there doing nothing. Even if I hit ^C it ignores.

What am I missing?

OSX El capitan
node - v7.0.0
npm - v3.10.8
grunt-cli - v1.2.0
grunt - v0.4.5

Help :)

**Comment by jamesmhaley** [comment]
It seemed another grunt task was blocking it. I'll try work out what it was.

**Comment by jamesmhaley** [comment]
So the issue was in the register task and calling that task the same name as the task i was trying to run

`grunt.registerTask('accessibility', ['accessibility']);`

Bad brain day. Hopefully this will help someone else :)
